[2024-05-25T22:20:01.035+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T22:20:01.058+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T22:19:00+00:00 [queued]>
[2024-05-25T22:20:01.065+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T22:19:00+00:00 [queued]>
[2024-05-25T22:20:01.066+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-25T22:20:01.078+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): first-step-el> on 2024-05-25 22:19:00+00:00
[2024-05-25T22:20:01.084+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1934) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-25T22:20:01.085+0000] {standard_task_runner.py:63} INFO - Started process 1936 to run task
[2024-05-25T22:20:01.086+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_catedra_***', 'first-step-el', 'scheduled__2024-05-25T22:19:00+00:00', '--job-id', '326', '--raw', '--subdir', 'DAGS_FOLDER/dag_clase.py', '--cfg-path', '/tmp/tmpwhhja6ac']
[2024-05-25T22:20:01.087+0000] {standard_task_runner.py:91} INFO - Job 326: Subtask first-step-el
[2024-05-25T22:20:01.124+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_catedra_airflow.first-step-el scheduled__2024-05-25T22:19:00+00:00 [running]> on host b2cfee8f0b2b
[2024-05-25T22:20:01.207+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_catedra_***' AIRFLOW_CTX_TASK_ID='first-step-el' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T22:19:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-25T22:19:00+00:00'
[2024-05-25T22:20:01.209+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T22:20:01.216+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T22:20:01.228+0000] {logging_mixin.py:188} INFO - Query completed
[2024-05-25T22:20:01.230+0000] {logging_mixin.py:188} INFO - execute_query_result_set()
[2024-05-25T22:20:01.236+0000] {logging_mixin.py:188} INFO - Connection to database was succesful
[2024-05-25T22:20:01.245+0000] {logging_mixin.py:188} INFO - Inserts performed wihout fail
[2024-05-25T22:20:01.246+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-25T22:20:01.246+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T22:20:01.254+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_catedra_***, task_id=first-step-el, run_id=scheduled__2024-05-25T22:19:00+00:00, execution_date=20240525T221900, start_date=20240525T222001, end_date=20240525T222001
[2024-05-25T22:20:01.299+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T22:20:01.316+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-25T22:20:01.320+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
